Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -275,6 +276,75 @@ extension VisualInstruction.Component: Codable {
         case lane
     }
     
+    /**
+     :nodoc:
+     Subtype that provides more context about the component guidance view that may help in visual
+     markup and display choices.
+     */
+    public enum SubType: String, Codable, CaseIterable {
```

Comment:
We’re avoiding the term “Type” in favor of “Kind”, as in `VisualInstruction.Component.Kind`, so “SubType” is a bit incongruous. If only a `guidanceView` can have a subtype, then call it `GuidanceViewKind`. Ideally, we’d make this an associated value of `Kind.guidanceView`, but that’s difficult because `Kind` is string-backed.

Reply:
It's hard to find the best option here because this option is not yet public. At the same time it seems that `SubType` is only related to guidance views, so I'll rename it to `GuidanceViewKind`.